app/src/main/res/values/strings.xml

```diff
@@ -43,4 +43,7 @@
     <string name="gender_other">Other</string>
     <string name="gender_undisclosed">Undisclosed</string>
 
+    <string name="suggestions">Suggestions</string>
+    <string name="suggestions_failed">Couldn\'t access suggestions</string>
```

Comment:
Nit: Can we rename this resource? There's probably going to be more than one error state that can occur on this page, so we'll want to be able to differentiate between the error messages if/when we need them.